lib/Doctrine/Annotations/Metadata/Builder/PropertyMetadataBuilder.php

```diff
@@ -56,7 +56,7 @@ public function withBeingDefault() : self
     }
 
     /**
-     * @param array<int|float|string|bool>|null $enum
+     * @param array<string, array<int|float|string|bool>|string> $enum
```

Comment:
Does this fail? String is not subtype of array...

Reply:
Same reason as above, this is for the `'literal' => '1,2,3'` part